eng/AfterSigning.targets

```diff
@@ -0,0 +1,12 @@
+<Project>
```

Comment:
How does this get called? This is a new file, so there can't be any references to it previously, and you didn't add one here.

Reply:
https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md#engaftersigningtargets-optional